src/main/java/seedu/stock/ui/HelpWindow.java

```diff
@@ -15,17 +16,72 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String ADD = "ADD \n---------------------------\n";
+    public static final String DELETE = "---------------------------\nDELETE\n---------------------------\n";
+    public static final String FIND = "---------------------------\nFIND\n---------------------------\n";
+    public static final String UPDATE = "---------------------------\nUPDATE\n---------------------------\n";
+    public static final String EXIT = "---------------------------\nEXIT\n---------------------------\n";
+
+    public static final String ADD_DESCRIPTION =
+            "format: \nadd n/<name> s/<source of stock> q/<quantity> l/<location in warehouse>\n\n";
+
+    public static final String DELETE_DESCRIPTION =
+            "format: \ndelete sn/<serial number>\n\n";
```

Comment:
Should we include an example which shows the usage of deleting multiple stock?

Reply:
Yes, I think it would be good to include deleting multiple stock at once.

Reply:
I agree!

Reply:
Noted I will include them into the usage!